packages/@lwc/integration-karma/test/component/LightningElement.render/index.spec.js

```diff
@@ -82,3 +83,11 @@ it('supports returning different templates', () => {
         expect(elm.shadowRoot.textContent).toBe('Template 2');
     });
 });
+
+it('throws an error when render() returns an invalid value', () => {
+    const elm = createElement('x-render-invalid', { is: RenderInvalid });
+    expect(() => document.body.appendChild(elm)).toThrowErrorDev(
+        Error,
```

Comment:
```suggestion
        Error,
```

You don't need to assert the `Error` class if you're just checking for a generic `Error`.

Reply:
https://github.com/salesforce/lwc/blob/175d7aa48697dc118df4a7fe62d1393284267612/packages/@lwc/integration-karma/README.md#L82

Should update this, then, at some point...

Reply:
Oops yeah, that's wrong. The message can be a regex or a string as well.

Reply:
Ak, I just realized you will need `toThrowCallbackReactionErrorDev` because of native lifecycle. You can see why detailed [here](https://github.com/salesforce/lwc/pull/3662#:~:text=fire%20as%20expected.-,New%20error%20behavior,-%E2%9A%A0%EF%B8%8F).